app/constants/import_constants.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ImportConstants
+  IMPORT_MAPPING = {
+    srg_id: 'SRGID',
+    stig_id: 'STIG ID',
+    rule_severity: 'Severity',
+    title: 'Requirement',
+    vuln_discussion: 'VulDiscussion',
+    status: 'Status',
+    check_content: 'Check',
+    fixtext: 'Fix',
+    vendor_comments: 'Vendor Comments',
```

Comment:
this field seems to be missing from some of the example files, causing an error as well

Reply:
The latest commit has this fixed.